content/post/2017-04-28-will-landau-s-portfolio-for-minimalists.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Will Landau's portfolio: a website for minimalists
```

Comment:
The title contains a colon, which is a special character in YAML, so the title has to be double quoted.